packages/cf-component-icon/src/svgs/api.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"><title>-</title><path d="M15.124,12.749v-9.5A1.826,1.826,0,0,0,13.343,1.47H2.657A1.826,1.826,0,0,0,.876,3.251v9.5A1.826,1.826,0,0,0,2.657,14.53H13.343A1.826,1.826,0,0,0,15.124,12.749Zm-9.26-2.493L5.507,9.069H3.667L3.31,10.256H2.123L3.9,5.151H5.21l1.781,5.105ZM5.27,8.178c-.356-1.068-.534-1.721-.594-1.84s-.059-.3-.119-.356c-.059.3-.3,1.009-.653,2.2H5.27Zm5.936-1.425A1.534,1.534,0,0,1,10.671,8a2.21,2.21,0,0,1-1.484.415H8.713V10.2H7.644V5.21h1.6a2.145,2.145,0,0,1,1.425.415,1.378,1.378,0,0,1,.534,1.128Zm-2.434.831h.356A1.346,1.346,0,0,0,9.9,7.407a.626.626,0,0,0,.238-.594.725.725,0,0,0-.178-.534A.955.955,0,0,0,9.306,6.1H8.831V7.585h-.06Zm3.384,2.671V5.21h1.068v5.105H12.155Z"/></svg>
```

Comment:
What's with the `<title>-</title>` tags?

Reply:
New icons have them removed.